app/controllers/application_controller.rb

```diff
@@ -90,7 +90,17 @@ def self.session_key_prefix
   # This will rescue any un-handled exceptions
   rescue_from StandardError, :with => :error_handler
 
+  def self.handle_exceptions?
+    Thread.current[:application_controller_handles_exceptions] != false
```

Comment:
At the moment, all request specs, upon encountering an exception while running the controller/view, will return a 500 response. In some places, that's what we want to happen, because we caused the error deliberately, to test that it correctly becomes a 500.

But the rest of the time, when it's indicative of a bug that the spec has caught, it's infinitely more useful for rspec to report an exception, with an exception name and a backtrace, instead of saying "got 500, expected 200".

So my plan is that eventually we should be able to flip the default on this, so "make sure error handling is working correctly" specs can turn on said error handling, and the rest of the specs can all run without the obfuscation of the safety net.

In this process, it's already been hugely helpful, upon encountering such a 500-for-200 failure, to drop in an `ApplicationController.handle_exceptions = false`, and immediately see the "real" problem.